src/components/structureDefSettings/StructureDefSettings.tsx

```diff
@@ -72,150 +100,105 @@ const StructureDefSettings: React.FC<StructureDefSettingsProps> = ({
-        helperText="A free text natural language description of the structure and its use."
-      />
-
-      <br />
-      <label htmlFor="url">url</label>
-      <input
-        type="url"
-        name="url"
-        value={((url as unknown) as string) || ""}
-        onChange={(e) => setUrl(e.target.value)}
-      />
-      <br />
-      <label htmlFor="id">id</label>
-      <input
-        type="text"
-        name="id"
-        value={id || ""}
-        onChange={(e) => setId(e.target.value)}
-      />
-      <br />
-      <label htmlFor="publisher">publisher</label>
-      <input
-        type="text"
-        name="publisher"
-        value={publisher || ""}
-        onChange={(e) => setPublisher(e.target.value)}
-      />
-      <br />
-      <label htmlFor="copyright">copyright</label>
-      <input
-        type="text"
-        name="copyright"
-        value={copyright || ""}
-        onChange={(e) => setCopyright(e.target.value)}
-      />
-      <br />
-      <label htmlFor="title">title</label>
-      <input
-        type="text"
-        name="title"
-        onChange={(e) => setTitle(e.target.value)}
-      />
-      <br />
-      <label htmlFor="status">Status</label>
-      <select
-        name="status"
-        onChange={(e) =>
-          setStatus(e.target.value as StructureDefinitionStatusKind)
-        }
-        defaultValue={status || ""}
-      >
-        <option value="active">active</option>
-        <option value="draft">draft</option>
-        <option value="retired">retired</option>
-        <option value="unknown">unknown</option>
-      </select>
-      <br />
-      <input
-        type="datetime-local"
-        onChange={(e) => {
-          const date = new Date(e.target.value);
-          setDate(
-            new Date(
-              date.getTime() - date.getTimezoneOffset() * 60000
-            ).toISOString()
-          );
-        }}
-      />
-      <br />
-      <label htmlFor="purpose">purpose</label>
-      <input
-        type="text"
-        name="purpose"
-        onChange={(e) => setPurpose(e.target.value)}
-      />
-      <br />
-      <input
-        type="checkbox"
-        name="abstract"
-        onChange={(e) => setAbstract(e.target.checked)}
-      />
-      <label htmlFor="abstract">abstract</label>
-      <br />
-      <input
-        type="checkbox"
-        name="experimental"
-        onChange={(e) => {
-          setExperimental(e.target.checked);
-        }}
-      />
-      <label htmlFor="experimental">experimental</label>
-      <br />
-      {renderContact}
-      <br />
-      <input
-        type="submit"
-        value="submit"
-        onClick={(e) => handleEditSettings(e)}
-      />
-    </form>
+    <Container className={classes.formContainer}>
+      <form className={classes.form}>
+        <InputWithHelpYouyou
+          label="Id"
+          value={id || ""}
+          tool="The version can be globally unique, or scoped by the Logical Id of the resource."
```

Comment:
For each of your 'tool' props, I see you are setting a string depending on the the input nature.
I think those would be better in a const file exporting a value like so for instance :

`const toolTips : {
   id : "The version can be globally unique, or scoped by the Logical Id of the resource.",
   url : "The identifier that is used to identify this structure when it is referenced in a specification, model, design or an instance. This URL is where the structure can be accessed.",
   name : [...]
   [...]
}`